topics/admin/tutorials/connect-to-compute-cluster/tutorial.md

```diff
@@ -0,0 +1,855 @@
+
+Once written, restart Galaxy with `sudo supervisorctl restart all`.
+
+**Part 4 - Verify***
+
+Run the **Multicore Tool* with various resource parameter selections:
+- Use default job resource parameters
+- Specify job resource parameters:
+  - 1 core
+  - 2 cores
+  - Some value for walltime from 1-24
+
+The cores parameter can be verified from the output of the tool. The walltime can be verified with `scontrol`:
+
+```console
+$ scontrol show job 24
+JobId=24 JobName=g24_multi_anonymous_10_0_2_2
+   UserId=galaxy(999) GroupId=galaxy(999)
+   Priority=4294901747 Nice=0 Account=(null) QOS=(null)
+   JobState=COMPLETED Reason=None Dependency=(null)
+   Requeue=1 Restarts=0 BatchFlag=1 Reboot=0 ExitCode=0:0
+   RunTime=00:00:05 TimeLimit=12:00:00 TimeMin=N/A
+   SubmitTime=2016-11-05T22:01:09 EligibleTime=2016-11-05T22:01:09
+   StartTime=2016-11-05T22:01:09 EndTime=2016-11-05T22:01:14
+   PreemptTime=None SuspendTime=None SecsPreSuspend=0
+   Partition=debug AllocNode:Sid=gat2016:1860
+   ReqNodeList=(null) ExcNodeList=(null)
+   NodeList=localhost
+   BatchHost=localhost
+   NumNodes=1 NumCPUs=1 CPUs/Task=1 ReqB:S:C:T=0:0:*:*
+   TRES=cpu=1,node=1
+   Socks/Node=* NtasksPerN:B:S:C=0:0:*:* CoreSpec=*
+   MinCPUsNode=1 MinMemoryNode=0 MinTmpDiskNode=0
+   Features=(null) Gres=(null) Reservation=(null)
+   Shared=OK Contiguous=0 Licenses=(null) Network=(null)
+   Command=(null)
+   WorkDir=/srv/galaxy/server/database/jobs/000/24
+   StdErr=/srv/galaxy/server/database/jobs/000/24/galaxy_24.e
+   StdIn=StdIn=/dev/null
+   StdOut=/srv/galaxy/server/database/jobs/000/24/galaxy_24.o
+   Power= SICP=0
+```
+
+Note that the `TimeLimit` for this job (which I gave a 12 hour time limit) was set to `12:00:00`.
+
+### So, what did we learn?
+
+Hopefully, you now understand:
+- The various ways in which tools can be mapped to destinations, both statically and dynamically
+- How to write a dynamic tool destination (DTD)
+- How to write a dynamic python function destination
+- How to use the job resource parameter selection feature
+
+### Further Reading
+
+- The [sample dynamic tool destination config file](https://github.com/galaxyproject/galaxy/blob/dev/config/tool_destinations.yml.sample) fully describes the configuration language
+- [Dynamic destination documentation](https://wiki.galaxyproject.org/Admin/Config/Jobs#Dynamic_Destination_Mapping)
+- Job resource parameters are not as well documented as they could be, but the [sample configuration file](https://github.com/galaxyproject/usegalaxy-playbook/blob/master/files/galaxy/test.galaxyproject.org/config/job_resource_params_conf.xml) shows some of the possibilities.
+- [usegalaxy.org's job_conf.xml](https://github.com/galaxyproject/usegalaxy-playbook/blob/master/templates/galaxy/usegalaxy.org/config/job_conf.xml.j2) is publicly available for reference.
```

Comment:
`https://github.com/galaxyproject/usegalaxy-playbook/blob/master/templates/galaxy/usegalaxy.org/config/job_conf.xml.j2` does not exist

Reply:
fixed